packages/api/src/utils/server/registerRoute.ts

```diff
@@ -3,13 +3,15 @@ import {ServerInstance, RouteConfig, ServerRoute} from "./types.js";
 export function registerRoute(
   server: ServerInstance,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  route: ServerRoute<any>
+  route: ServerRoute<any>,
+  namespace?: string
 ): void {
   server.route({
     url: route.url,
     method: route.method,
     handler: route.handler,
-    schema: route.schema,
+    // append the namespace as a tag for downstream consumption of our API schema, eg: for swagger UI
+    schema: {...route.schema, ...(namespace ? {tags: [namespace]} : undefined)},
```

Comment:
I'd suggest we also set the `operationId`, as this is more inline with the beacon-API

```suggestion
    schema: {...route.schema, ...(namespace ? {tags: [namespace]} : undefined), operationId: route.id},
```

Changes routes
- before `/documentation/static/index.html#/beacon/get_eth_v1_beacon_genesis`
- after `/documentation/static/index.html#/beacon/getGenesis`